### ISSUE TITLE
Fundamentals of Neural Networks

### ISSUE THREAD

**Issue by mraymer** [mraymer/data-science-independent-study#6]
- [ ] Perceptrons, MLPs and feed-forward neural networks
- [ ] Back-propagation and network training
- [ ] Optimization via stochastic gradient descent
- [ ] Optimization meta-parameters: ADAM, etc.
